network/transport/grpc/connection_manager.go

```diff
@@ -57,17 +57,19 @@ type fatalError struct {
 }
 
 func (s fatalError) Error() string {
-	return s.error.Error()
+	// use Sprintf in case error is nil
+	return fmt.Sprintf("%s", s.error)
```

Comment:
Geeft rare string. Kan behoorlijk zoeken zijn waar vandaan komt ook in logs. Beter gepaste melding.

```go
if s.error != nil {
        return "nil subject in fatal connection management error!"
}
```